_sub/compute/k8s-kiam/vars.tf

```diff
@@ -89,6 +89,11 @@ variable "chart_version" {
   type        = string
   description = "KIAM helm chart version"
   default     = null
+
+  validation {
+    condition     = tonumber(substr(var.chart_version, 0, 1)) >= 6
```

Comment:
This wont work with versions greater than single digit

Reply:
Presume a split function call might be a good idea?  Should turn the chart version into a list containing major, minor and patch versions.  Then the comparison can just be done on the major portion.